composer.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "mollie/magento2",
   "description": "Mollie Payment Module for Magento 2",
-  "version": "1.2.1",
```

Comment:
Version is required for Magento Marketplace

Reply:
Ah, I was afraid of that, thanks.